runtime/protocol/http/common/src/aws/smithy/kotlin/runtime/http/endpoints/util/Util.kt

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+package aws.smithy.kotlin.runtime.http.endpoints.util
```

Comment:
perhaps some of this should live in `../endpoints/internal` package rather than `util` package?